tests/app/forms/test_questionnaire_form.py

```diff
@@ -283,6 +283,99 @@ def test_date_combined_range_too_large_validation(self):
                              schema.error_messages['DATE_PERIOD_TOO_LARGE'] % dict(max='50 days'))
 
 
+    def test_date_mm_yyyy_combined_single_validation(self):
+        with self.test_request_context():
```

Comment:
I know we have a test for invalid period limits in a combined schema, should we add one for `mm-yyyy` also?

Reply:
Do you mean like `test_date_mm_yyyy_combined_range_too_large_validation` below?

Reply:
No as in the 5xx when the schema is invalid in the combined case.

Reply:
like `test_invalid_minimum_period_limit_and_single_date_periods`

Reply:
Probably for the best. Strictly speaking this hasn't changed the way that works  but lets be sure it hasn't.